openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -33,6 +34,14 @@
         )}
     % endif
 
+    % if ENABLE_DASHBOARD_PORTFOLIO_PROJECT_UPSELL_MODAL:
```

Comment:
As noted elsewhere, I'm hoping this could just use `INCLUDE_PORTFOLIO_UPSELL_MODAL`.
Also, you need `.is_enabled()`.